kubernetes/generic/bookie.yaml

```diff
@@ -0,0 +1,158 @@
+#
+# Copyright 2016 Yahoo Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+    name: bookie-config
+data:
+    PULSAR_MEM: "\" -Xms512m -Xmx512m -XX:MaxDirectMemorySize=1g\""
+    dbStorage_writeCacheMaxSizeMb: "256" # Write cache size (direct memory)
+    dbStorage_readAheadCacheMaxSizeMb: "256" # Read cache size (direct memory)
+    zkServers: zk-0.zookeeper,zk-1.zookeeper,zk-2.zookeeper
+    statsProviderClass: org.apache.bookkeeper.stats.PrometheusMetricsProvider
+---
+
+## BookKeeper servers need to access the local disks and the pods
+## cannot be moved across different nodes.
+## For this reason, we run BK as a daemon set, one for each node in the
+## cluster, unless restricted by label selectors
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+    name: bookie
+    labels:
+        app: pulsar
+        component: bookkeeper
+spec:
+    template:
+        metadata:
+            labels:
+                app: pulsar
+                component: bookkeeper
+                # Specify cluster to allow aggregation by cluster in
+                # the metrics
+                cluster: us-central
+            annotations:
+                prometheus.io/scrape: "true"
+                prometheus.io/port: "8000"
+
+        spec:
+            containers:
+              - name: bookie
+                image: streamlio/pulsar:latest
```

Comment:
Or we can keep `streamlio/pulsar*` everywhere and we can mention into ReadMe about [cluster](https://github.com/merlimat/pulsar/blob/7dc068a06fa85da201687d5afd96aee220e9ef15/docker/Dockerfile), [grafana](https://github.com/merlimat/pulsar/blob/7dc068a06fa85da201687d5afd96aee220e9ef15/docker/grafana/Dockerfile), [dashboard](https://github.com/merlimat/pulsar/blob/7dc068a06fa85da201687d5afd96aee220e9ef15/dashboard/Dockerfile) location and customization in case one wants to change port.